### PR TITLE
Mock API example improvement

### PR DESCRIPTION
**Pull request by panpan2**
- Corrected the javascript imports of the example 
- Added an additional line to demonstrate better how to use the mock  secure worker

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -275,13 +275,15 @@ To use mock mode of operation, you have to provide a function which resolves
 script names to their content. For example:
 
 ```javascript
-import fs from 'fs';
-import path from 'path';
-import SecureWorker from 'secureworker';
+var fs = require('fs');
+var path = require('path');
+const SecureWorker = require('secureworker')
```
Comment (mitar) [inline README.md]: `;` at th end.

Comment (mitar) [reply README.md]: But i do think these lines are really required. This is just a style question. So I would prefer not changing them (then you do not need to transpile in most node versions).

Comment (panpan2) [reply README.md]: Correct me if I'm wrong but I think that the way the imports are in the example, they are not suitable for use in nodejs (es6 imports are not available in nodejs). Everyone using this package would need to change them to "require".

Comment (mitar) [reply README.md]: Oh, I misread this. Yes, this is better.

**Pull request merged**